descriptions/css/properties/border.json

```diff
@@ -0,0 +1,9 @@
+{
+  "css": {
+    "properties": {
+      "border": {
+        "__short_description": "The <strong><code>border</code></strong> <a href='https://developer.mozilla.org/docs/CSS'>CSS</a> property sets an element's border. It's a <a href='https://developer.mozilla.org/docs/Web/CSS/Shorthand_properties'>shorthand</a> for <a href='https://developer.mozilla.org/docs/Web/CSS/border-width'><code>border-width</code></a>, <a href='https://developer.mozilla.org/docs/Web/CSS/border-style'><code>border-style</code></a>, and <a href='https://developer.mozilla.org/docs/Web/CSS/border-color'><code>border-color</code></a>."
+      }
```

Comment:
"The border CSS property sets an element's border. It's a shorthand for..." -> "The border shorthand CSS property sets an element's border." ?

We haven't listed the components of other shorthands, so I'd suggest not doing so here  (with structured content we would ideally represent shorthands explicitly so a tool can pull those and present them in a useful way).

Reply:
Good idea. I'll fix this up going forward. Should I drop this text from only the structured data (i.e., use the seoSummary to exclude this, but include it in the first paragraph on the wiki) or from the wiki entirely?

Reply:
Good question. Let's keep it in the Wiki and just exclude it from short-descriptions.